src/endpoints/v1.es6.js

```diff
@@ -144,9 +155,20 @@ class APIv1Endpoint {
     });
   }
 
+  log (state, method, uri, options, status, err, duration) {
+    if (this.config.debugLevel === 'info') {
```

Comment:
Right now, it's all string based - I'll make a separate PR whereupon I overhaul the logging system.